sdk/provisioning/Azure.Provisioning.CloudMachine/tests/Azure.Provisioning.CloudMachine.Tests.csproj

```diff
@@ -3,7 +3,8 @@
     <LangVersion>12</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\core\Azure.Core.TestFramework\src\Azure.Core.TestFramework.csproj" />
+    <ProjectReference Include="..\..\..\..\sdk\core\Azure.Core.TestFramework\src\Azure.Core.TestFramework.csproj" />
     <ProjectReference Include="..\src\Azure.Provisioning.CloudMachine.csproj" />
+    <PackageReference Include="System.Text.Json" VersionOverride="9.0.0-rc.1.24431.7"/>
```

Comment:
Was this needed for a specific feature or ...?

Reply:
this is needed as the previous version (8.0) is flagged as having a vulnerability